### ISSUE TITLE
Add github actions as CI option

### ISSUE THREAD

**Issue by mellery451** [ripple/rippled#3224]
investigate adding github CI as an alternative to travis.

**Comment by scottschurr** [comment]
@ximinez, are you working on this?  If so, could you assign it to yourself or remove it as a duplicate?  Thanks.

**Comment by ximinez** [comment]
Yes. https://github.com/ximinez/rippled/actions?query=branch%3Agithub-ci

**Comment by mathbunnyru** [comment]
We now use GitHub CI
